packages/cmf-webpack-plugin/package.json

```diff
@@ -4,7 +4,7 @@
   "main": "dist/index.js",
   "mainSrc": "src/index.js",
   "scripts": {
-    "prepublish": "npm run build && npm run test",
+    "prepare": "npm run build && npm run test",
```

Comment:
yarn or npm, we need to choose 😇

Reply:
I guess yarn is wider used now in packages, no?

Reply:
in fact yarn is mandatory to work on talend/ui, since we have a yarn.lock
would you suggest to replace all the `npm run` in scripts ?